### ISSUE TITLE
conda environment on CGD systems currently limits python versions to 3.7.0

### ISSUE THREAD

**Issue by ekluzek** [ESCOMP/CTSM#1792]
The conda environment on CGD systems currently limits the python versions to 3.7.0. You can technically ask for other python versions, but you won't get them without extra work. Also note that on other systems using python 3.7.0 seems to be problematic, but using python 3.7.9 seems to work fine. So until this is fixed on CGD systems, we should use a different file for the conda environment on CGD systems from other systems.

**Comment by ekluzek** [comment]
ctsm5.1.dev100 handles this by having a separate file for CGD systems. The right long term solution is for this to be resolved in the conda version used in CGD. Once that is done we should be able to close this for good. For now I'm marking this as a "wontfix".

**Comment by ekluzek** [comment]
I'm working with Joseph O'Neil to get an updated version of conda on at least izumi. This new version should resolve this issue for good.
